core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -135,6 +136,10 @@ abstract class TaskContext extends Serializable {
   @DeveloperApi
   def taskMetrics(): TaskMetrics
 
+  /** ::DeveloperApi:: */
```

Comment:
This special tag is only necessary if there is a java doc (please add one). The javadoc here will be incorrectly displayed on the docs page.